docs/getting_started.md

```diff
@@ -31,7 +31,6 @@ Wash is distributed as a single binary; the only prerequisite is `libfuse`. Here
     * If anything fails, then check out the [known issues page]({{ '/known_issues' | relative_url }}) to see if the failure(s) correspond to any of the known issues. Otherwise, please don't hesitate to ask us on [slack](https://puppetcommunity.slack.com/app_redirect?channel=wash) for help (or file an [issue](https://github.com/puppetlabs/wash/issues))! Note that you can use `wash --verify-install` to test any fixes.
 
 * You're good to go! Try running `wash` to start up the shell[^1]
-    * If you plan on extending Wash, then check out the [external plugin docs]({{ '/docs/external-plugins' | relative_url }}).
```

Comment:
Removed this b/c I already include something like this in an error message for a first-time user who does not enable any of the core plugins.

Reply:
I don't feel like they're mutually exclusive.

Reply:
It's unnecessary redundancy. A first-time, external plugin-only user is going to start the shell, going to say no to all the core plugins, and are then going to see the external plugin error message which already points to the docs.